src/databricks/labs/ucx/queries/assessment/main/00_6_count_table_storage_types.sql

```diff
@@ -0,0 +1,20 @@
+-- viz type=table, name=Table counts by storage, columns=Storage,count
+-- widget title=Table counts by storage, row=1, col=4, size_x=2, size_y=4
+SELECT storage, COUNT(*) count
+FROM (
+SELECT
+       CASE
+           WHEN STARTSWITH(location, "dbfs:/mnt") THEN "DBFS MOUNT"
+           WHEN STARTSWITH(location, "/dbfs/mnt") THEN "DBFS MOUNT"
+           WHEN STARTSWITH(location, "dbfs:/databricks-datasets") THEN "Databricks Demo Dataset"
+           WHEN STARTSWITH(location, "/dbfs/databricks-datasets") THEN "Databricks Demo Dataset"
+           WHEN STARTSWITH(location, "dbfs:/") THEN "DBFS ROOT"
+           WHEN STARTSWITH(location, "/dbfs/") THEN "DBFS ROOT"
+           WHEN STARTSWITH(location, "wasb") THEN "UNSUPPORTED"
```

Comment:
Why do you skip unsupported? It'll give a false impression to dashboard user

Reply:
Removed the filter condition.